lesson04/assignment/index.html

```diff
@@ -9,7 +9,31 @@
 </head>
 <body>
   <main class="main">
-
+    <header>
```

Comment:
You will want to put your header and footer outside of your main. Your main is for the core content of your page. Any code that is shared across other pages (like navbar, header, footer) can go outside the main https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main

Reply:
Perfect! Thank you so much I was having trouble finding documentation on it.